### DISCUSSION TITLE
fleetDM not showing my ip address unless i manual query them

### DISCUSSION THREAD

**Discussion by samocolorado** [Security-Onion-Solutions/securityonion#11181] [Unsupported Versions] [answered]
Hello,

I'm facing an issue with my network setup. I'm using the IP range 192.0.0.0/24 internally, "i know its used for public IPs but in my case its in my local Network and i cant change them ". 
This is causing problems when I install the osquery agent (specifically the fleetdm agent) on devices within this IP range. When the agent is installed, it successfully retrieves information like the OS name, but the private and public IP fields are left empty (---) in the interface. I'm only able to see the IP addresses if I run a query.

Interestingly, when I install the same agent on hosts with a regular private IP range like 10.10.1.0/24, everything works fine. The private IP is displayed on the home page. However, this isn't the case for hosts in the 192.0.0.0/24 range.

I'm looking for a way to change the configuration so that the agent properly displays the IP addresses (e.g., 192.0.0.2) on the home page. Could you please guide me to the configuration file where I can make this change?

Best regards,

**Comment by defensivedepth** [comment, marked as answer]
This is an issue specific to FleetDM. I would suggest you post on their support forums: https://fleetdm.com/support

Their Slack channel is very active.
